J0190UIL-1215 Clarify optimal size in MWT documentation (#284)
MicroEJ/docs: ApplicationDeveloperGuide/UI/MWT/how-to-widget.rst

Unless the widget is using an ``OptimalDimension`` in its style, the actual size of the widget will most likely be bigger than the optimal size returned in this method.

The ``size`` parameter of the ``computeContentOptimalSize()`` method initially contains the size available for the widget.
An available width or height equal to ``Widget.NO_CONSTRAINT`` means that the optimal size should be computed without considering any restriction on the respective axis.
Before the method returns, the size object should be set to the optimal size of the widget.

When implementing this method, the ``getStyle()`` method may be called in order to retrieve the style of the widget.

For example, the following snippet computes the optimal size of an image widget:

.. code-block:: Java

	@Override
	protected void computeContentOptimalSize(Size size) {
		size.setSize(this.image.getWidth(), this.image.getHeight());
	}

Rendering the content of the widget
~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

The ``renderContent()`` method is called by the MWT framework in order to render the content of the widget.

> The ``g`` parameter can be used in order to draw the content of the widget with the translation and clipping area which matches the widget's bounds.

Reviewer: ```suggestion
The ``g`` parameter is used to draw the content of the widget.
It is already configured with the translation and clipping area which matches the widget's bounds.
```

The ``renderContent()`` method is called by the MWT framework in order to render the content of the widget.

The ``g`` parameter is used to draw the content of the widget.

> It is already configured with the translation and clipping area which matches the widget's bounds.

Reviewer: ```suggestion
It is already configured with the translation and clipping area which match the widget's bounds.
```
